butane_core/src/many.rs

```diff
@@ -125,23 +125,30 @@ where
         Ok(())
     }
 
-    /// Loads the values referred to by this many relationship from the
+    /// Query the values referred to by this many relationship from the
     /// database if necessary and returns a reference to them.
-    pub fn load(&self, conn: &impl ConnectionMethods) -> Result<impl Iterator<Item = &T>> {
+    pub fn query(&self) -> Result<Query<T>> {
+        let owner: &SqlVal = match &self.owner {
+            Some(o) => o,
+            None => return Err(Error::NotInitialized),
+        };
+        Ok(T::query().filter(BoolExpr::Subquery {
+            col: T::PKCOL,
+            tbl2: self.item_table.clone(),
+            tbl2_col: "has",
+            expr: Box::new(BoolExpr::Eq("owner", Expr::Val(owner.clone()))),
+        }))
+    }
+
+    /// Loads the values referred to by this many relationship from a
+    /// database query if necessary and returns a reference to them.
+    pub fn load_query(
```

Comment:
The `query`/`load_query` methods have a couple rough edges
1. The returned query already has the filter set. Any attempts to set the filter will result in a query that's no longer actually associated with the `Many`.
2. `load_query` assumes that they query provided was returned from `query` and that it does not do any further filtering of the results (if it did, always adding in the unsaved items would be invalid since they might not match the filter).

I don't think it's necessary to remove these rough edges for now until there's actually a clear use case for these methods being more flexible, but it would be nice to document the expectations in the method comments. Alternatively these two methods could stay private for now, with `load_ordered` the only new public method.

Not blocking.